components/eventing-controller/testing/v2/matchers/eventmeshsub/matchers.go

```diff
@@ -0,0 +1,38 @@
+package eventmeshsub
```

Comment:
Does it make sense to delete this file and move it contents to `eventing-controller/testing/v2/matchers.go`?

Reply:
I added a separate file isolate the matchers for eventMeshSubscriptions. Otherwise, in `eventing-controller/testing/v2/matchers.go`, we have methods like `HaveEvent` and by name you cannot identify is it for kyma subscription or eventMesh subscription.